test/libsolidity/syntaxTests/inlineAssembly/invalid/transient_storage_assignment.sol

```diff
@@ -0,0 +1,10 @@
+contract test {
+    uint transient x;
+    function f() public {
+        assembly {
+            x := 2
+        }
+    }
+}
+// ----
+// TypeError 1408: (95-96): Only local variables are supported. To access storage variables, use the ".slot" and ".offset" suffixes.
```

Comment:
Might as well add a mention for transient storage variables here as well.

Reply:
True. But maybe here we could change to `state variables` ? It seems that the meaning here is about that rather than data location.

Reply:
Changed to `state variables`.